Build Server is claiming there's a compile error

```
/home/vsts/work/1/s/src/main/java/twilightforest/item/MazeMapItem.java:44: error: method createMapData in class MazeMapItem cannot be applied to given types;
		createMapData(itemstack, level, worldX, worldZ, scale, trackingPosition, unlimitedTracking, level.dimension(), worldY);
		^
  required: ItemStack,Level,int,int,int,boolean,boolean,ResourceKey<Level>,int,boolean
  found:    ItemStack,Level,int,int,byte,boolean,boolean,ResourceKey<Level>,int
  reason: actual and formal argument lists differ in length
/home/vsts/work/1/s/src/main/java/twilightforest/item/MazeMapItem.java:76: error: cannot find symbol
		mapData.ore = ore;
		^
  symbol:   variable mapData
  location: class MazeMapItem
```

--- a/src/main/java/twilightforest/item/MazeMapItem.java
+++ b/src/main/java/twilightforest/item/MazeMapItem.java
@@ -41,7 +41,7 @@ public class MazeMapItem extends MapItem {
 
 	public static ItemStack setupNewMap(Level level, int worldX, int worldZ, byte scale, boolean trackingPosition, boolean unlimitedTracking, int worldY, boolean mapOres) {
 		ItemStack itemstack = new ItemStack(mapOres ? TFItems.FILLED_ORE_MAP.get() : TFItems.FILLED_MAZE_MAP.get());
-		createMapData(itemstack, level, worldX, worldZ, scale, trackingPosition, unlimitedTracking, level.dimension(), worldY);
+		createMapData(itemstack, level, worldX, worldZ, scale, trackingPosition, unlimitedTracking, level.dimension(), worldY, mapOres);
 		return itemstack;
 	}
 
@@ -73,7 +73,7 @@ public class MazeMapItem extends MapItem {
 
 		TFMazeMapData mapdata = new TFMazeMapData(scaledX, scaledZ, (byte) scale, trackingPosition, unlimitedTracking, false, dimension);
 		mapdata.calculateMapCenter(level, x, y, z); // call our own map center calculation
-		mapData.ore = ore;
+		mapdata.ore = ore;
 		TFMazeMapData.registerMazeMapData(level, mapdata, getMapName(i)); // call our own register method
 		stack.getOrCreateTag().putInt("map", i);
 		return mapdata;

